cellprofiler/utilities/sharedmethods.py

```diff
@@ -0,0 +1,33 @@
+"""
+Shared methods used by multiple modules
+"""
+
+import numpy
+import skimage.morphology
+
+
+def dilation(x_data, structuring_element):
+
```

Comment:
I understand you didn't introduce this formatting, but do you mind fixing it? There shouldn't be a newline after `def` and `if`/`elif`/`else`. 

Additionally, `elif` and `else` are unnecessary after a `return` statement.